code/modules/networks/computer3/peripherals.dm

```diff
@@ -959,9 +969,7 @@
 		func_tag = "CLOWN_ID_SCANNER"
 		clownifies_card = 1
 
-		return_badge()
-			var/dat = "<font face='Comic Sans MS'>Card: <a href='?src=\ref[src];card=1'>[src.authid ? "Eject" : "-----"]</a></font>"
-			return dat
+			// this used to use comic sans here, but i havent found a way to change the font of just one part
```

Comment:
You'd have to do this conditionally on the frontend, wrap the text in a `Box` and change the font based on whether is is clown or not

Reply:
did it